deepola/wake/examples/tpch_polars/main.rs

```diff
@@ -120,6 +122,8 @@ pub fn get_query_service(
         "q23" => q23::query(table_input, output_reader),
         "q24" => q24::query(table_input, output_reader),
         "q25" => q25::query(table_input, output_reader),
+        "q26" => q26::query(table_input, output_reader),
+        "q27" => q27::query(table_input, output_reader),
```

Comment:
Can we comment after these extended queries (q23 ...) about what they refer to (e.g. `// q1, progressiveDB`)? Maybe here or `mod` above

Reply:
Just as a quick reference without going into each file to read the comment